employee.rb

```diff
@@ -0,0 +1,60 @@
+require './reviews'
+
+class Employee
+
+  attr_reader :name, :salary, :reviews
+
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def add_review(text)
+    @reviews << text
+  end
+
+  def rate_satisfactory
+    @satisfactory = true
+  end
+
+  def rate_unsatisfactory
+    @satisfactory = false
+  end
+
+  def satisfactory?
+    return @satisfactory
+  end
+
+  def give_raise_by_percent(percent)
+    starting_salary = @salary
+    @salary = (percent + 1) * starting_salary
+  end
+
+  def give_raise_with_dollars(dollars)
+    starting_salary = @salary
+    @salary = dollars + starting_salary
+  end
+
+  def performance_rating_based_on_review(review)
+    bad_score = []
+    good_score = []
+    bad_things = ["but", "not", "room for improvement", "difficult", "confusion", "less", "negative", "not", "lack", "inadequate", "limitation", "concern", "longer", "interrupt", "disagree", "off"]
+    good_things = ["asset", "pleasure", "quick", "willing", "help", "success", "happy", "responsive", "consistent", "effective", "satisfied", "impressed", "enjoy", "devoted", "perfect", "ownership", "positive", "encourage", "well"]
+    review.split.each do |word|
+      if bad_things.include?(word)
+        bad_score << word
```

Comment:
Cool.  Calling this bad_score probably makes you think it's a number.  you should probably call it something like bad_words_found or something like that to make it clear that it's an array.